userguide/one_shot_object_detection/README.md

```diff
@@ -62,18 +61,59 @@ To learn more about deploying One-Shot to CoreML details, refer to the [Object D
 
 ## How it works
 
-In this section, we will go into detail on what happens under the hood of the `create` method. An OSOD model is trained in two steps: First, we use the starter images to generate synthetic data and annotations, and second, we train an object detector.
+In this section, we will go into detail on what happens under the hood of the `create` method. A One-Shot Object Detector model is trained in two steps: First, we use the starter images to generate synthetic data and annotations, and second, we train a model on the generated synthetic data.
 
 **Stage 1: Automatic synthetic data generation**
 
-To begin training, a set of synthetic training data needs to be generated using the provided starter image(s). 
+To begin training, the provided starter image(s) are used to generate a set of synthetic training data. 
 The synthetic images are generated by applying yaw, pitch, and roll rotations on a set of background images. If no background images are provided, a set of default background images are automatically downloaded and used.
 
 **Stage 2: Train the Object Detector model**
 
 To learn about the training implementation details, refer to the [Object Detector: How it Works](https://apple.github.io/turicreate/docs/userguide/object_detection/how-it-works.html) chapter.
 
+## Troubleshooting
+
+##### My starter image can appear upside down as well as rotated 90 degrees in the wild. Is there anything I can do to improve the quality of the model?
+
+To ensure a higher quality model, instead of providing *one* starter image, feel free to provide *more than one* starter images. As of Turi Create 5.8, for every starter image, we rotate the starter image by up to 20 degrees. As a result, to detect both inverted and erect projections of the same starter image, you can provide *2* starter images with the same label. Here is an example to rotate a Stop Sign starter image by 90 degrees, 180 degrees, and 270 degrees, and construct an SFrame with four starter images:
+
+```python
+def pil_to_tc(pil_image):
+    image = np.array(pil_img)
+    FORMAT_RAW = 2
+    return tc.Image(_image_data=image.tobytes(),
+                    _width=image.shape[1],
+                    _height=image.shape[0],
+                    _channels=image.shape[2],
+                    _format_enum=FORMAT_RAW,
+                    _image_data_size=image.size)
+
+import turicreate as tc
+image_path = ... set image path here ...
```

Comment:
Please make this code more readable and copy passable. Use nice function names, variable names, and potentially refactor to make it nicer.